server/sonar-server/src/main/java/org/sonar/ce/settings/ProjectSettingsFactory.java

```diff
@@ -17,33 +17,29 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.server.setting;
+package org.sonar.ce.settings;
 
-import java.util.List;
 import org.sonar.api.ce.ComputeEngineSide;
 import org.sonar.api.config.Settings;
-import org.sonar.api.server.ServerSide;
-import org.sonar.db.property.PropertiesDao;
-import org.sonar.db.property.PropertyDto;
+import org.sonar.db.DbClient;
 
-@ServerSide
+// FIXME move to CE
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)